schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+  id            INT GENERATED ALWAYS AS IDENTITY,
+  name     	    VARCHAR(250),
+  date_of_birth       date,
+  escape_attempts       INT,
+  neutered           BOOLEAN,
+  weight_kg          DECIMAL(10,2),
```

Comment:
hello there @1GORDON the weight should be a decimal not a boolean because that won't be possible and not an integral because it has some decimal value,